src/Components/AlbumCard/AlbumCard.js

```diff
@@ -1,57 +1,80 @@
 import { GET_SINGLE_ALBUM }from '../../queries';
 import { useQuery } from '@apollo/client';
-import { Skeleton } from '@material-ui/lab';
+
 import './AlbumCard.css'
 import { Link } from 'react-router-dom'
+import { useEffect } from 'react';
+// import CoverStandIn from '../../../public'
 
-function AlbumCard ({ title }) {
-  
-  const { loading, error, data } = useQuery(GET_SINGLE_ALBUM, {
-    variables: { title: title }
-  })
+
+function AlbumCard ({ album }) {
+
+  // const [currentAlbum, setCurrentAlbum] = useState()
+
+
+  // useEffect()
+  // const { loading, error, data } = useQuery//(GET_SINGLE_ALBUM, {
+  //   variables: { title: title }
+  // })
   
-  if (loading) return (
-    <div className='card_card' data-cy='card_card'>
-      <Skeleton animation="wave" variant="rect" width={250} height={250} />
-      <Skeleton animation="wave" variant="text" height={32} />
-      <Skeleton animation="wave" variant="text" />
-      <Skeleton animation="wave" variant="text" />
-      <Skeleton animation="wave" variant="text" />
-    </div>
-  )
-  if (error) return `Error! ${error.message}`
+  // if (loading) return (
+  //   <div className='card_card' data-cy='card_card'>
+  //     <Skeleton animation="wave" variant="rect" width={250} height={250} />
+  //     <Skeleton animation="wave" variant="text" height={32} />
+  //     <Skeleton animation="wave" variant="text" />
+  //     <Skeleton animation="wave" variant="text" />
+  //     <Skeleton animation="wave" variant="text" />
+  //   </div>
+  // )
+  // if (error) return `Error! ${error.message}`
 
-  const displayGenres = (list) => {
-   return list.map(genre => <p className='card_genre' data-cy='card_genre'>{genre}</p>)
-  }
+  // const displayGenres = (list) => {
+  //   console.log(list)
+  //  return list.map(genre => <p className='card_genre' data-cy='card_genre'>{genre}</p>)
+  // }
+
+  // const displayArtists = (list) => {
+  //   return list.map(name => <p>{ name }</p>)
+  //  }
+
+
+   const displayImage = (image) => {
```

Comment:
Just ensuring that this commented code is either removed or activated where necessary